File: MPI/mpicheck/tests/functions/1.0/MPI_Testall.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       INTEGER var_1(10)
       LOGICAL var_2
       INTEGER var_3(10,10)
       INTEGER var_4
        call mpi_testall(var_0, var_1, var_2, var_3, var_4)
       call pmpi_testall(var_0, var_1, var_2, var_3, var_4)
        end program main
    

File: MPI/mpicheck/tests/functions/1.0/MPI_Testall.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    int var_0;
    MPI_Request var_1[2];
    int *var_2;
    MPI_Status var_3[2];
    int ret;
    /* calls */
    ret = MPI_Testall(var_0, var_1, var_2, var_3);
    ret = PMPI_Testall(var_0, var_1, var_2, var_3);
    return 0;
}
